packages/components/src/organisms/RatingSummary/RatingDistributionItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { forwardRef, type HTMLAttributes } from 'react'
+import Icon from '../../atoms/Icon'
+
+export interface RatingDistributionItemProps
+  extends HTMLAttributes<HTMLLIElement> {
+  ratingIndex: number
```

Comment:
I think we need to add a ts-doc here. This prop will be the current star, right ? I mean: 1,2,3,4 or 5.